tests/vmi_multus_test.go

```diff
@@ -1076,6 +1098,16 @@ func runSafeCommand(vmi *v1.VirtualMachineInstance, command string) error {
 	}, 15)
 }
 
+func getInterfaceNameByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string) (string, error) {
```

Comment:
`virtClient` seems to be unused.

Reply:
oh right. missed that when removing the vmi != nil assertion. thank you for the catch.
Done.